README.md

```diff
@@ -368,6 +381,36 @@ c2patool sample/C.jpg trust \
 
 **Note:** This sample image should show a `signingCredential.untrusted` validation status since the test signing certificate used to sign them is not contained on the trust lists above.
 
+## Adding a manifest to fragmented BMFF content
+
+The ISO base media file format (BMFF) is a container file format that defines a structure for files that contain time-based multimedia data such as video and audio.
+
+Add a manifest to a fragmented BMFF file by using the `fragment` subcommand, as follows:
+
+```
+c2patool <PATH | PATTERN> fragment [--fragments_glob]
+```
+
+Where `<PATTERN>` is a [glob pattern](https://en.wikipedia.org/wiki/Glob_(programming)).
+
+For example, to add manifest to a video file:
+
+```
+c2patool -m test2.json -o  /1080p_out \
+  /Downloads/1080p/video/avc1/init.mp4 \ 
```

Comment:
Let's shorten this example to be:

"c2patool -m test2.json -o  /1080p_out \
  /Downloads/1080p/avc1/init.mp4 \
fragment --fragments_glob "seg-*[0-9].m4s"

I just removed "video" from the sample path.

Reply:
Done